manifest.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "Start | Stop",
+  "description": "Assign or un-assign yourself from an issue.",
+  "commands": {
+    "start": {
+      "ubiquibot:example": "/start",
```

```suggestion
      "ubiquity:example": "/start",
```

I want to deprecate "ubiquibot" across all of our code and marketing materials in favor of new terminology. I think "ubiquity" is the safest and will always be relevant so that is my default here. 

However as a heads up I was thinking of referring to the system as "UbiquityOS" or "superintendent" in reference to the kernel.